CMakeLists.txt

```diff
@@ -120,6 +120,25 @@ if ((ENABLE_HIP) AND (NOT ENABLE_KOKKOS))
   list(APPEND RAJA_PERFSUITE_DEPENDS blt::hip_runtime)
 endif()
 
+#
+# Are we using Caliper
+#
+set(RAJA_PERFSUITE_USE_CALIPER off CACHE BOOL "")
```

Comment:
Is there a reason to have a different USE_CALIPER for each project?  Caliper regions+data are composable across projects, so I wonder if there is a use case where we would want to enable caliper in one project but not another - if not, keeping it the same across projects may be nice.

Reply:
One rationale for the different flag was the fact that Caliper support is not in BLT, where then it could be made general, including the way the flag propagates across subprojects.